### ISSUE TITLE
Templated inclusion

### ISSUE THREAD

**Issue by StoneCypher** [StoneCypher/fsl#489]
It's not entirely clear how to do this, but, it should be able to construct a state machine from other state machines as pieces.

This is not the same as heirarchality / harel #7 - that's about not repeating yourself, and domain ownership.

This is also not the same as embeddable machine fragments #235 - that's a one time inclusion.

What we'd like here would be something like

1. Define a finite state machine representing some re-usable cog
1. Give it a way to express names that have an externalizable variation
    1. String prefix?
    1. Namespace would be cleaner, and could be a supremely gross string prefix under the hood
1. Say "this is <grid>auth, this is <mail>auth", &c

Thus the inclusion can be repeated without identity ambiguity

They'd need to be nestable

**Comment by StoneCypher** [comment]
Blocked by #648

**Comment by StoneCypher** [comment]
Duplicate of #1352.

#1352 (v11 source-level modularity: cross-file imports + parameterized machine templates) is exactly namespaced repeatable inclusion.

Dispositioned in the #1849 legacy-milestone sweep.
